src/test/java/net/sf/jabref/importer/fileformat/MedlinePlainImporterTest.java

```diff
@@ -198,6 +198,14 @@ public void testWithNbibFile() throws IOException, URISyntaxException {
     }
 
     @Test
+    public void testWithMultipleEntries() throws IOException, URISyntaxException {
+        Path file = Paths
+                .get(MedlinePlainImporter.class.getResource("MedlinePlainImporterStringOutOfBounds.txt").toURI());
+        List<BibEntry> entries = importer.importDatabase(file, Charset.defaultCharset()).getDatabase().getEntries();
```

Comment:
When the bib file is in UTF8, then please set the Charset explicit to UTF8, too. `DefaultCharsets.UTF8` (or similar), otherwise test will fail on windows in gradle.